pkg/volume/cinder/cinder_util.go

```diff
@@ -135,10 +139,26 @@ func (util *CinderDiskUtil) DeleteVolume(cd *cinderVolumeDeleter) error {
 	return nil
 }
 
-func (util *CinderDiskUtil) CreateVolume(c *cinderVolumeProvisioner) (volumeID string, volumeSizeGB int, err error) {
+func GetZonesFromNodes(kubeClient clientset.Interface) (sets.String, error) {
```

Comment:
Does this need to be exported outside this file?  (ie: can we make it lowercase `getZones...` ?)